core/extension/gdextension_interface.h

```diff
@@ -403,6 +427,7 @@ typedef struct {
 
 	GDExtensionScriptInstanceCall call_func;
 	GDExtensionScriptInstanceNotification notification_func;
+	GDExtensionScriptInstanceNotification2 notification_func2;
```

Comment:
I failed to notice this in my previous review, but adding a field to `GDExtensionScriptInstanceInfo` will also cause compatibility issues. We'll need to do something similar to what has already been done in this PR for `GDExtensionClassCreationInfo`, including adding a new struct and a new function (maybe `script_instance_create2`?) to accept it.

Sorry for not catching this earlier!

Reply:
In the latest version I believe that I have addressed this: https://github.com/godotengine/godot/compare/36d8b2d8dc68fd8a3d1d83be589a12642d53506f..c4537d7d693f17fed41329d8bd86e4c4f36cd61e
Had to rebase the PR because of a merge conflict.

Reply:
Thanks! I haven't tested it, but skimming the code, this looks good to me :-)

I guess we just need to resolve on the GDExtension team the question of leaving the deprecated property on the struct in the interface.